### DISCUSSION TITLE
Hide/show/reorder plugins

### DISCUSSION THREAD

**Discussion by aolko** [Moonvy/figmaEX#5] [Ideas]
Please add the ability to hide/show plugins from the list (and only show not hidden ones), that will be useful to the people with >20 plugins installed. And reordering (dran-n-drop) would also be nice, with an option to reset the plugin order to default.

**Comment by yArna** [comment]
Good idea

Before that, now there is a trick where you can put the important plugins in front by sorting
Then hide the plugins by adjusting the height of the plugin sidebar

<img src="https://user-images.githubusercontent.com/82231420/115108739-b2506c00-9fa4-11eb-846b-2aafa03b2174.gif" width="300"></img>

**Reply by aolko** [reply]
that's currently more of a dirty hack
